src/components/SignUpForm/styles.scss

```
@@ -0,0 +1,37 @@
+@import "../../styles/main.scss";
+
+.container {
+    min-height: 100vh;
+    @include flex-center();
+}
+
+.signUp-form {
+    margin: auto;
+    height: rem(400px);
+    width: rem(650px); 
+    flex-direction: column;
+    
+    background-color: $background;
+  
+    @include flex-center();
+}
+
+@mixin button-bg($background) {
+  background-color: $background;
+  margin-top: 16px;
+  &:hover {
+    background-color: darken($background, 8%);
+  }
+}
+
+.btn {
+  color: white;
+  text-decoration: none;
+  padding: 5px 10px;
+  border-radius: 3px;
+}
+
+.btn-submit {
+  @include button-bg(#5a6570);
+}
+    
```

```suggestion
.sign-up-form {
  margin: auto;
  height: rem(400px);
  width: rem(650px);
  flex-direction: column;
  background-color: $background;
  @include flex-center();

  &__submit-button {
    color: $light;
    text-decoration: none;
    padding: 5px 10px;
    border-radius: 3px;
    @include button-bg(#5a6570);
  }
}
```
1. if you are really sure to make a reusable @mixin, then please move this to styles/_mixins.scss for best practice.
2. Please make sure there are no empty spaces
3. Class `btn` and `btn-submit` could me moved inside `sign-up-form`. 
Ref: https://github.com/Cobda/cobda-web/pull/9/files

Any suggestion for class naming? @LilBank

---

I replied here: https://github.com/Cobda/cobda-web/pull/7#discussion_r563290999